src/cluster.h

```diff
@@ -38,8 +38,11 @@ namespace zim
   class Cluster : public std::enable_shared_from_this<Cluster> {
       typedef std::vector<offset_t> Offsets;
 
+    public:
```

Comment:
> It would be better to be coherent. Either we use accessors all the time or we accept that const field can be directly read.

Agree. However, the follow-up changes introducing incremental decompression will eliminate the controversy with the public `compression` field and the `getCompression()` accessor.

I don't see any problem with `isExtended` remaining public.

Reply:
Ok